sccp/figures/figureThomsonDrop.py

```diff
@@ -99,7 +100,10 @@ def plotDifferentialExpression(
         X, X2 = np.array(origX.uns["Pf2_A"]), np.array(sampledX.uns["Pf2_A"])
         all_r2, all_r2_2 = (
             [linregress(X[:, i], numberOfCellType)[2] ** 2 for i in range(X.shape[1])],
-            [linregress(X2[:, i], numberOfCellType)[2] ** 2 for i in range(X.shape[1])],
+            [
+                linregress(X2[:, i], numberOfCellType)[2] ** 2
+                for i in range(X2.shape[1])
```

Comment:
Since they are the same shape it doesn't make a difference in function but is just more understandable